CMakeLists.txt

```diff
@@ -170,6 +170,30 @@ if(LIBPRESSIO_HAS_OPENMP)
     )
 endif()
 
+option(LIBPRESSIO_HAS_DIGIT_ROUNDING "build the DIGIT ROUNDING plugin" OFF)
+if(LIBPRESSIO_HAS_DIGIT_ROUNDING)
+  set(LIBPRESSIO_COMPRESSORS "${LIBPRESSIO_COMPRESSORS} DR")
+  find_package(dround REQUIRED)
+  find_package(zlib REQUIRED)
+  target_sources(libpressio
+    PRIVATE
+      ${CMAKE_CURRENT_SOURCE_DIR}/src/plugins/compressors/digit_rounding_plugin.cc
+    )
+  target_link_libraries(libpressio PRIVATE dround zlib)
+endif()
+
+option(LIBPRESSIO_HAS_BIT_GROOMING "build the BIT GROOMING plugin" OFF)
+if(LIBPRESSIO_HAS_BIT_GROOMING)
+  set(LIBPRESSIO_COMPRESSORS "${LIBPRESSIO_COMPRESSORS} BG")
+  find_package(bg REQUIRED)
+  find_package(zlib REQUIRED)
```

Comment:
the package name is ZLIB.  See the [cmake docs](https://cmake.org/cmake/help/v3.15/module/FindZLIB.html?highlight=zlib%20zlib)